cal/cal/models.py

```diff
@@ -550,7 +550,7 @@ def __str__(self):
     def hours(self, calendar_ids=None, start=None, end=None):
         events = self.get_events(calendar_ids=calendar_ids, start=start, end=end)
 
-        return EventCollection(lambda: events).total_time() / 3600
+        return round(float(EventCollection(lambda: events).total_time()) / 3600, 2)
```

Comment:
This is necessary to round to two decimal places. Before the pie chart was rounded down, but this becomes sort of confusing when you have pie charts per week and all your hours are off by 30 mins or something.